src/ol_superset/pythonpath/superset_config.py

```diff
@@ -260,3 +262,24 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 SLACK_API_TOKEN = vault_client.secrets.kv.v2.read_secret(
     path="app-config", mount_point="secret-superset"
 )["data"]["data"]["slack_token"]
+
+#######################
+# Custom Jinja Macros #
+#######################
+# Enable usage of current_user_email() macro until Superset releases that feature
+
+
+def current_user_email() -> Optional[str]:
+    """
+    Get the email (if defined) associated with the current user.
+
+    :returns: The email
+    """
+
+    try:
+        return g.user.email
+    except Exception:  # pylint: disable=broad-except
+        return None
+
+
+JINJA_CONTEXT_ADDONS = {"current_user_email": current_user_email()}
```

Comment:
```suggestion
JINJA_CONTEXT_ADDONS = {"current_user_email": current_user_email}
```
We want to set the function object in the context. Adding the parentheses will set the result of the function call.